rpc/namespaces/ethereum/eth/api.go

```diff
@@ -951,7 +956,6 @@ func (e *PublicAPI) GetTransactionReceipt(hash common.Hash) (map[string]interfac
 		"transactionHash": hash,
 		"contractAddress": nil,
 		"gasUsed":         hexutil.Uint64(parsedTx.GasUsed),
-		"type":            hexutil.Uint(txData.TxType()),
```

Comment:
why is this being removed?

Reply:
to imitate geth response, since geth doesnt return `type` on `getTransactionReceipt`